### DISCUSSION TITLE
Is it possible to reopen pull requests?

### DISCUSSION THREAD

**Discussion by b1n4ryj4n** [microsoft/winget-pkgs#122294] [General] [answered]
Hi there 

is it possible to reopen Pull requests? Ghostscript has a security vulnerability and needs the update [CVE-2023-43115](https://github.com/advisories/GHSA-9p55-888j-qxrh)

https://github.com/microsoft/winget-pkgs/pull/119663#issuecomment-1750590253

**Comment by mdanish-kh** [comment, marked as answer]
You can open up a new pull request to add that package.

As for the previous versions, we should probably be removing them from the community repo

Edit: I've opened PRs to remove the previous versions from the repository

**Reply by b1n4ryj4n** [reply]
Unfortunately I can not add the new version because the silent installation was removed in version [10.01.0](https://artifex.com/news/ghostscript-10.01.0-disabling-silent-install-option).

> All tools must support a silent install to be permitted in the Windows Package Manager Community Repository. If you have an executable that does not support a silent install, then we cannot provide that tool at this time.
